opm/parser/eclipse/EclipseState/Eclipse3DProperties.cpp

```diff
@@ -18,7 +18,7 @@
 
 
 #include <boost/algorithm/string/join.hpp>
-
+#include <opm/common/OpmLog/OpmLog.hpp>
```

Comment:
OK - why is this? If the `Eclipse3DProperties`has calls to `OpmLog` those as well should be replaced with adding messages to a container; and the `Eclipse3DProperties` class should also get a `MessageContainer` instance.